src/backend/routers/student.ts

```diff
@@ -64,6 +64,42 @@ export const student = router({
       return result;
     }),
 
+  /**
+   * Adds a new IEP for the given student.
+   */
+  editIep: authenticatedProcedure
+    .input(
+      z.object({
+        student_id: z.string(),
+        start_date: z.date(),
+        end_date: z.date(),
+      })
+    )
+    .mutation(async (req) => {
+      const { student_id, start_date, end_date } = req.input;
+      const { userId } = req.ctx.auth;
+
+      // Check if the student exists and if the case manager is assigned to the student
+      const existingStudent = req.ctx.db
```

Comment:
I think you might need to also add `.execute()` to run this method.

Reply:
Good call!